src/docker-osx-dev

```diff
@@ -846,6 +853,21 @@ function initial_sync {
   log_debug "Creating parent directories in Docker VM: $ssh_cmd"
   $DOCKER_HOST_SSH_COMMAND "$ssh_cmd"
 
+  local readonly excludes=($EXCLUDES)
+  exclude_flags=${excludes[@]/#/--exclude }
+  local readonly includes=($INCLUDES)
+  include_flags=${includes[@]/#/--include }
+  for key in "${!parent_dirs[@]}"; do
```

Comment:
exclamation point returns the keys/indices of the array instead of the values. I need the keys, not the values, because I'm looping through 2 arrays at the same time.

`for local key in...`? that's not valid syntax. for-variables are always local, I think